src/main/java/EigenValue.java

```diff
@@ -0,0 +1,59 @@
+public class EigenValue {
```

Comment:
EigenValue is the projects name. As class EigenValue suggest that it is just a number type, not something that can calculate eigenValue.

Either it is a Matrix that has the method of calculation or it is a class that can do arithmetics.